### ISSUE TITLE
IconButton does not work as shown in documentation

### ISSUE THREAD

**Issue by zbcoding** [elcharitas/chakra-ui-svelte#49]
https://chakra-ui.com/docs/components/icon-button/usage

Should be `<IconButton aria-label='Search database' icon={<SearchIcon />} />` as shown in chakra documentation.
But this will cause a svelte parse-error in svelte template

**Comment by elcharitas** [comment]
Hello @zbcoding, Thanks for creating your first issue here! We're thrilled to have you on board and greatly appreciate your contribution.

It's important to note that Svelte differs from React in some key ways. Specifically, you cannot pass a node directly to a Svelte prop. Instead, you can pass a component, which would look something like this:

```html
<IconButton aria-label='Search database' icon={SearchIcon} />
```

Additionally, please be aware that **chakra-ui-svelte** is still in development and may not be completely stable. As a result, you might encounter occasional discrepancies or unexpected behavior. We encourage you to reach out and report any issues you come across, as this helps us improve the library for everyone.

Thank you once again for your valuable input, and we look forward to your continued contributions! 😊
